src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -127,13 +127,6 @@ object ScalafixPlugin extends AutoPlugin {
       Invisible
     )
 
-  private val scalafixCompletions: SettingKey[ScalafixCompletions] =
```

Comment:
inlining this saves some setting keys (sbt-scalafix adds a lot...)

Reply:
This caused a regression, see https://github.com/scalacenter/sbt-scalafix/pull/405